### ISSUE TITLE
Cannot run rake tasks in Dockerfile when including exception_handler

### ISSUE THREAD

**Issue by wkhatch** [richpeck/exception_handler#83]
I've revisited this gem again, after unsuccessfully trying it in the past but having now upgraded the application to rails 5.2. Now I'm hitting a new issue, in that, when building a docker image, my Dockerfile has to precompile the assets as part of the docker image build. This gem is forcing a db connection during initialize, so that fails, as I don't have a db connection during the docker build.

**Comment by richpeck** [comment]
Hi William, many thanks for taking the time to contact me and use ExceptionHandler.

In terms of your request, do you have the `db` option set at all?

**Comment by wkhatch** [comment]
Yes, and I hadn't considered that. I could disable it for production, although I'd rather not. I'll play around with it more and see if I can figure out a work around. Thanks.
